process-dir.js

```diff
@@ -230,46 +327,49 @@ async function process (dir) {
   sources = sources.filter(([file, source]) => {
     if (/(^|\/|\\)index(\.browser|\.native)?\.js/.test(file)) {
       return true
-    } else if (/(^|\n)export /.test(source)) {
+    }
+    if (/(^|\n)export /.test(source)) {
       return false
-    } else {
-      let fixed = file.replace(/\.js$/, sep + 'index.js')
-      throw error(`Rename ${file} to ${fixed}`)
     }
+
+    let fixed = file.replace(/\.js$/, `${sep}${INDEX_JS}`)
+    throw error(`Rename ${file} to ${fixed}`)
   })
-  let files = sources.map(([file]) => file)
+
+  let files = []
 
   await Promise.all(
     sources.map(async ([file, source]) => {
-      if (file.endsWith('index.browser.js')) {
-        await replaceToESM(dir, file, source)
-      } else if (file.endsWith('index.native.js')) {
-        await replaceToESM(dir, file, source)
-      } else {
-        await Promise.all([
-          replaceToCJS(dir, file, source),
-          replaceToESM(dir, file, source),
-          replacePackage(dir, file, files)
-        ])
+      let pathToCurrentFile = join(dir, file)
+      files.push(file)
+
+      switch (true) {
```

Comment:
Ideas of using `switch(true)` and replacing literals look nice

Reply:
I saw it here:
https://twitter.com/LoremIpsumN/status/1326833240179666944?s=20